src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx

```diff
@@ -424,30 +473,13 @@ Here are the key metrics that are stored indefinitely.
   </tbody>
 </table>
 
-### Raw metric data
+### Metric raw data
 
-Raw metric data, from APM, Prometheus remote-write API, metrics functionality of the Telemetry SDK, and integrations built on the Telemetry SDK, is handled a little differently. Raw sub-minute metric data is stored for 30 days. Beyond 30 days, 1-minute resolution roll-up events are created and retained for 13 months. As with Metric timeslice data, we meter and bill on the size of the raw ingest, not the subsequent rollups.
+Metric raw data is handled differently for:
+- APM
+- Prometheus remote-write API
+- Metrics functionality of the Telemetry SDK
+- Integrations built on the Telemetry SDK
+Raw sub-minute metric data is stored for 30 days. Beyond 30 days, 1-minute resolution roll-up events are created and retained for 13 months. Like metric timeslice data, we meter and bill on the size of the raw ingest, not the subsequent rollups.
```

Comment:
This first sentence is odd. If you're listing the different entities, I wouldn't describe how the data is stored at the beginning of the bullet, I'd leave just the Raw sub-minute metric data. Then explain how it is stored.